tests/chainer_tests/test_function_node.py

```diff
@@ -915,10 +913,10 @@ def setUp(self):
             self._init_ones(self._to_grad_names(self.y_names))
             self.gys = None
 
-    def use_gpu(self):
+    def use_device(self, device):
         for value in six.itervalues(self.__dict__):
             if isinstance(value, chainer.Variable):
-                value.to_gpu()
+                value.to_device(backend.get_device(device))
```

Comment:
`get_device` is redundant.

```suggestion
                value.to_device(device)
```